src/ontology/mondo-edit.obo

```diff
@@ -585022,6 +585026,21 @@ is_a: MONDO:0004981 {source="PMID:38033089"} ! atrial fibrillation
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7886" xsd:anyURI
 
+[Term]
+id: MONDO:1030015
+name: acquired porphyria
+def: "A porphyria disorder that is acquired during the lifetime of the individual." [MONDO:patterns/acquired]
+xref: GARD:10353 {source="MONDO:mondoIsNarrowerThanSource"}
+xref: ICD9:277.1 {source="MONDO:equivalentTo"}
```

Comment:
Sorry I missed it previously. According to the file found [here](https://www.cms.gov/medicare/coding-billing/icd-10-codes/icd-9-cm-diagnosis-procedure-codes-abbreviated-and-full-code-titles#:~:text=Version%2032%20Full%20and%20Abbreviated%20Code%20Titles%20%C2%A0%E2%80%93%20Effective%20October%201%2C%202014%20(ZIP)), this ICD9 code is not specific to the acquired disease

Reply:
I removed it from here and added it to 'porphyrin metabolism disease', which is appropriate based on the UMLS xref already on that term.